### PR TITLE
chore(FDS-489): [Monorepo] - bump prettier to v2.5.1

### PR DESCRIPTION
**Pull request by lordpixel**
This bumps prettier to its most recent version v2.5.1.

### Dependencies
`package.json` file updated because of the version bump.

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

File: ADR/prettier.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Prettier
```
Comment (odiseo42) [inline ADR/prettier.md:1]: Is there any reason to keep this ADR? I see that the upgrade was successful without any implications. If there's nothing to discuss in the future I would say there's no need to have an ADR, or even delete the existing ADR if the decision or technical problem discussed in the document has already been solved.

**Review by odiseo42 — APPROVED** [review]
LGTM other than the existential question on the ADR doc

File: ADR/prettier.md (hunk 1 above)
Comment (lordpixel) [reply ADR/prettier.md:1]: This is actually an error on my side, thanks for catching it. Let me add the AD record. 👍

Comment (lordpixel) [reply ADR/prettier.md:1]: Actually, I think it's important to keep this as a record of when was the last time we updated and if there were any important things to mention. What do you think, should we keep it?

**Pull request merged**